runc.go

```diff
@@ -21,6 +21,15 @@ import (
 // Format is the type of log formatting options avaliable
 type Format string
 
+// TopBody represents the structured data of the full ps output
+type TopBody struct {
```

Comment:
Body is a little off to me, maybe something about TopResults or something better?